src/Generator/InterfaceGenerator.php

```diff
@@ -34,8 +34,10 @@ public static function fromReflection(ClassReflection $classReflection)
         $cg->setSourceContent($cg->getSourceContent());
         $cg->setSourceDirty(false);
 
-        if ($classReflection->getDocComment() != '') {
-            $cg->setDocBlock(DocBlockGenerator::fromReflection($classReflection->getDocBlock()));
+        $docBlock = $classReflection->getDocBlock();
+
+        if ($docBlock) {
```

Comment:
Same here regarding explicitly checking for not false or the class. Your choice, not a blocker.

Reply:
Yeah, rolling with a simple conditional, since this is `false|object`